tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/APIView/APIViewRevisionTool.cs

```diff
@@ -0,0 +1,216 @@
+using System.CommandLine;
+using System.ComponentModel;
+using Azure.Sdk.Tools.Cli.Commands;
+using Azure.Sdk.Tools.Cli.Models;
+using Azure.Sdk.Tools.Cli.Models.Responses;
+using Azure.Sdk.Tools.Cli.Services;
+using ModelContextProtocol.Server;
+
+namespace Azure.Sdk.Tools.Cli.Tools.APIView;
+
+[McpServerToolType]
+[Description("APIView revision operations including comments and content")]
+public class APIViewRevisionTool : MCPMultiCommandTool
+{
+    // Sub-command constants
+    private const string GetComments = "get-comments";
+    private const string GetContent = "get-content";
+
+    public override CommandGroup[] CommandHierarchy { get; set; } = [SharedCommandGroups.APIView];
+
+    private static readonly string[] validContentTypes = ["Text", "CodeFile"];
+
+    private readonly IAPIViewService _apiViewService;
+    private readonly ILogger<APIViewRevisionTool> _logger;
+
+    private readonly Option<string> environmentOption = new("--environment")
+    {
+        Description = "The APIView environment (defaults to production)",
+        DefaultValueFactory = _ => "production"
+    };
+
+    private readonly Option<string> outputFileOption = new("--output-file"){Description = "Output file path to save the revision content"};
+    private readonly Option<string> contentReturnTypeOption = new("--content-return-type")
+    {
+        Description = "The APIView revision content type (text or codefile). Defaults to 'text'.",
+        DefaultValueFactory = _ => "text"
+    };
+
+    private readonly Option<string> apiViewUrlOption = new("--url"){Description = "The full APIView URL"};
```

Comment:
Can you update the description to make it clear that this is the "URL to the API review in APIView" Current description made me thinking `why do you ask for APIView URL when you know productio0n and staging URL?` APIView URL for me is home page URL.

Reply:
Also mark this as required param so you don't have to worry about checking if is passed or not.